package.json

```diff
@@ -47,6 +50,8 @@
     "redux-form": "^7.0.1",
     "redux-immutable-state-invariant": "^2.0.0",
     "redux-logger": "^3.0.6",
+    "redux-mock-store": "^1.2.3",
+    "redux-persist": "^4.8.3",
```

Comment:
Andi, thought you pulled this out into a separate PR that you then closed anyway? Are we bringing it back?